groups/restrictions.yaml

```diff
@@ -140,6 +140,7 @@ restrictions:
       - "^k8s-infra-staging-releng-test@kubernetes.io$"
       - "^k8s-infra-staging-publishing-bot@kubernetes.io$"
       - "^k8s-infra-staging-bom@kubernetes.io$"
+      - "^k8s-infra-staging-testgrid-exporter@kubernetes.io$"
```

Comment:
This group restriction name does not match the one in groups/sig-release/groups.yaml any more. I propose we keep `k8s-infra-staging-` as prefix and shorten the rest of the name.

Reply:
Thank you! 

I will rename `k8s-infra-staging-testgrid-exporter` to `k8s-infra-staging-tg-exporter` (or do you suggest a different name?)